oslabs-beta/Kubiquity
# BE server, Router, Controller

WHAT DOES THIS PR DO? 
1. Configured errorRouter and errorController to send back a JSON object of the event logs

HOW TO TEST THIS PR:
1. Run Docker Desktop and in the terminal run `minikube start`
2. Create a mongodb instance and create a env file with the URI OR paste it in the URI field in the errorModel.js file (remember to clear the field after testing)
3. Run `npm install` and `nodemon server/server.js`
4. Go to localhost:3000/error to see all the event logs

ADDITIONAL NOTES:
This is what an event log should look like within the array:
{
  "_id": "60e46ee7888d2a1a14e10e6e",
  "namespace": "default",
  "lastSeen": "30m",
  "type": "Normal",
  "reason": "Created",
  "object": "pod/alertmanager-prometheus-kube-prometheus-alertmanager-0",
  "message": "Created container config-reloader",
  "createdAt": "2021-07-06T14:55:35.126Z",
  "updatedAt": "2021-07-06T14:55:35.126Z",
  "__v": 0
}
